parm/config/gfs/config.resources

```diff
@@ -1062,10 +1063,18 @@ case ${step} in
     ntasks=3
     threads_per_task=1
     tasks_per_node=1
-    memory="20G"
-    if [[ "${CASE}" == "C768" || "${CASE}" == "C1152" ]]; then
+    case "${CASE}" in
+      "C768")
         memory="80GB"
-    fi
+	;;
+      "C1152")
+        memory="400GB"
+	walltime="01:20:00"
```

Comment:
Thats quite a jump from 20 mins to 1hr 20 mins, and 80GB to 400GB!  These probably are machine specific and should go in `config.resources.$MACHINE`

Reply:
What other machines have these been run on C1152?  I don't know if this is machine specific or just what it takes. 

Again, not all of these configurations were optimal.  They are "what works".  If things need to be at an optimal configuration to be merged, perhaps we should consider pausing this PR as that's a lot more work to get to that point.